common/src/main/java/com/wynntils/models/cosmetics/type/WynntilsCapeLayer.java

```diff
@@ -9,7 +9,7 @@
 import com.mojang.math.Axis;
 import com.wynntils.core.components.Managers;
 import com.wynntils.core.components.Models;
-import com.wynntils.features.players.WynntilsCosmeticsFeature;
+import com.wynntils.features.embellishments.WynntilsCosmeticsFeature;
```

Comment:
Wait, why are we referencing features from models?! :-O

Reply:
We can only register these at startup, and can't "de-register". So we return when the feature is disabled..